pkgs/development/mobile/androidenv/examples/shell.nix

```diff
@@ -6,12 +6,15 @@
     url = "https://github.com/NixOS/nixpkgs/archive/20.09.tar.gz";
     sha256 = "1wg61h4gndm3vcprdcg7rc4s1v3jkm5xd7lw8r2f67w502y94gcy";
   }),
-  pkgs ? import nixpkgsSource {},
-  pkgsi686Linux ? import nixpkgsSource { system = "i686-linux"; },*/
+  pkgs ? import nixpkgsSource {
+      config.allowUnfree = true;
```

Comment:
we are using 2 spaces to intend nix code

Reply:
Fixed in [d1a1f2f](https://github.com/NixOS/nixpkgs/pull/208380/commits/d1a1f2fa06fac055a57b28f3e15db7f8a793574c) on https://github.com/NixOS/nixpkgs/pull/208380